base/file.jl

```diff
@@ -88,19 +88,26 @@ end
 mkdir(path::AbstractString, mode::Signed) = throw(ArgumentError("mode must be an unsigned integer; try 0o$mode"))
 mkpath(path::AbstractString, mode::Signed) = throw(ArgumentError("mode must be an unsigned integer; try 0o$mode"))
 
-function rm(path::AbstractString; recursive::Bool=false)
+function rm(path::AbstractString; force::Bool=false, recursive::Bool=false)
     if islink(path) || !isdir(path)
         @windows_only if (filemode(path) & 0o222) == 0; chmod(path, 0o777); end # is writable on windows actually means "is deletable"
-        unlink(path)
+        try
+            unlink(path)
+        catch err
+            if force && isa(err, UVError) && err.prefix=="unlink" && err.code==Base.UV_ENOENT
+                return
+            end
+            systemerror(:unlink, true, extrainfo=path)
```

Comment:
what does the normal uverror from rm look like here? should we be rethrowing the error code too?